src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -322,7 +322,14 @@ const loadJobs = (specifiedVc) => {
     'select': {
       style: 'multi',
     },
+    'search': {
+      regex: true,
```

Comment:
Shell we open a new PR to discuss this? At least it should be recorded to changelog separately, so that user may be confused when they are searching jobs including dots which are legal in job names.